app/assets/javascripts/components/story/EstimateBadge/StoryEstimate.js

```diff
@@ -3,16 +3,16 @@ import PropTypes from 'prop-types';
 
 const estimateRule = (estimate) => estimate || '-';
```

Comment:
I know is not you code but for me, this is unnecessary since you have a default value by `defaultProps`.
```diff
- <span className='Story__estimated-value'>{estimateRule(estimate)}</span>
+ <span className='Story__estimated-value'>{estimate}</span>
```
This should work fine. Can you check?

Reply:
Oh sure, Thanks nolleto!